src/client/Microsoft.Identity.Client/Internal/Requests/ManagedIdentityAuthenticationRequest.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.ApiConfig.Parameters;
+using Microsoft.Identity.Client.Cache.Items;
+using Microsoft.Identity.Client.Core;
+using Microsoft.Identity.Client.Extensibility;
+using Microsoft.Identity.Client.Instance;
+using Microsoft.Identity.Client.ManagedIdentity;
+using Microsoft.Identity.Client.OAuth2;
+using Microsoft.Identity.Client.Utils;
+
+namespace Microsoft.Identity.Client.Internal.Requests
+{
+    internal class ManagedIdentityAuthenticationRequest : RequestBase
```

Comment:
nit: Our other request classes dont use the word Authentication. But not that big of a deal.

Reply:
Changed it to ManagedIdentityAuthRequest but open to suggestion for naming